prov/util/src/util_cq.c

```diff
@@ -468,6 +468,23 @@ int ofi_cq_cleanup(struct util_cq *cq)
 	return 0;
 }
 
+static int util_cq_control(struct fid *fid, int command, void *arg)
+{
+	struct util_cq *cq = container_of(fid, struct util_cq, cq_fid.fid);
+
+	switch (command) {
+	case FI_GETWAIT:
+		if (!cq->wait)
+			return -FI_ENODATA;
+		return fi_control(&cq->wait->wait_fid.fid, FI_GETWAIT, arg);
+	default:
+		if (!cq->wait)
```

Comment:
You can drop this if statement (but keep the error message -- though maybe convert to FI_WARN).

Reply:
Done